.gitignore

```diff
@@ -1,2 +1,5 @@
-.vscode/settings.json
-Notes/scripts/.idea/*
+.vscode/*
+/settings.json
+Notes/
+scripts/
+.idea/*
```

Comment:
Git wykrył .idea , powinien byc na nowej linii, wcześniej nie był wykrywany przez to że była ustawiona scieżka absolutna, obecnie powinien ignorować .idea oraz wszystkie pod-foldery :)